sensor/mapping_based_calibrator/src/calibration_mapper.cpp

```diff
@@ -403,22 +434,24 @@ void CalibrationMapper::checkKeyframeLost(Frame::Ptr keyframe)
   const Frame::Ptr & left_frame = data_->keyframes_[keyframe->keyframe_id_ - 1];
   Frame::Ptr & right_frame = keyframe;
 
-  Eigen::Affine3f dpose1(two_left_frame->pose_.inverse() * left_frame->pose_);
-  Eigen::Affine3f dpose2(left_frame->pose_.inverse() * right_frame->pose_);
+  Eigen::Affine3f delta_pose1(two_left_frame->pose_.inverse() * left_frame->pose_);
+  Eigen::Affine3f delta_pose2(left_frame->pose_.inverse() * right_frame->pose_);
   Eigen::Affine3f left_pose(left_frame->pose_);
   Eigen::Affine3f right_pose(right_frame->pose_);
 
-  auto d1 = dpose1.translation().normalized();
-  auto d2 = dpose2.translation().normalized();
+  auto d1 = delta_pose1.translation().normalized();
+  auto d2 = delta_pose2.translation().normalized();
 
   float trans_angle_diff = (180.0 / M_PI) * std::acos(d1.dot(d2));
-  trans_angle_diff =
-    dpose2.translation().norm() > parameters_->new_keyframe_min_distance_ ? trans_angle_diff : 0.0;
+  trans_angle_diff = delta_pose2.translation().norm() > parameters_->new_keyframe_min_distance_
```

Comment:
The name of translation is sometimes called trans or transl (in other calibrators), might be better the unify the name.
For me, "translation" would be the best one to understand.

Reply:
Addressed in [a23cea2](https://github.com/tier4/CalibrationTools/pull/148/commits/a23cea234f232fbd7c9b4a31c2976518f4e83918) and 
56bd2f0